scripts/vaccination_alerts/send-emails.ts

```diff
@@ -0,0 +1,206 @@
+import _ from 'lodash';
+import admin from 'firebase-admin';
+import { getFirestore } from '../common/firebase';
+import EmailService, {
+  EmailSendData,
+  isInvalidEmailError,
+} from '../alert_emails/email-service';
+import * as yargs from 'yargs';
+import {
+  DEFAULT_ALERTS_FILE_PATH,
+  readVaccinationAlerts,
+  generateEmailData,
+} from './utils';
+import { RegionVaccinePhaseInfo } from '../../src/cms-content/vaccines/phases';
+
+const BATCH_SIZE = 20;
+
+class SendVaccinationAlertsService {
+  db: FirebaseFirestore.Firestore;
+  emailService: EmailService;
+
+  invalidEmailCount: number = 0;
+  errorCount: number = 0;
+  emailSent: number = 0;
+
+  uniqueEmailAddress: { [email: string]: number } = {};
+  locationsWithEmails: { [fips: string]: number } = {};
+
+  constructor(db: FirebaseFirestore.Firestore, emailService: EmailService) {
```

Comment:
The abbreviated way to do this would be to drop the `db` / `emailService` field definitions above and do:

`constructor(private readonly db: FirebaseFirestore.Firestore, private readonly emailService: EmailService) { }`

But also see my other comment about maybe removing these as constructor arguments, since we always pass the same values.